src/main/java/com/dev/theater/model/MovieSession.java

```diff
@@ -0,0 +1,64 @@
+package com.dev.theater.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
```
in this case we have ManyToOne, because we have one Movie and many MovieSession in theater